diffrax/brownian/path.py

```diff
@@ -41,13 +46,20 @@ def __init__(
         self,
         shape: Union[Tuple[int, ...], PyTree[jax.ShapeDtypeStruct]],
         key: "jax.random.PRNGKey",
+        levy_area: str = "",
     ):
         self.shape = (
             jax.ShapeDtypeStruct(shape, jax.dtypes.canonicalize_dtype(None))
             if is_tuple_of_ints(shape)
             else shape
         )
         self.key = key
+        if levy_area not in ["", "space-time"]:
```

Comment:
Yes, you are right. The issue was that I was simultaneously working on adding space-time-time, and I merged some changes from the branch that was further ahead, and forgot to remove this reference to "space-time-time".
As things stand now, there should be no more references to "space-time-time" in docstrings that will be displayed in the docs, but the code is structured with "space-time-time" in mind, because I don't think it's worth reverting all the changes just to add them back in later. Still I made sure to delete all the code that is "space-time-time" only. I hope that's okay.

Reply:
That sounds perfect!